presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/HivePartitionName.java

```diff
@@ -101,12 +112,13 @@ public boolean equals(Object o)
 
         HivePartitionName other = (HivePartitionName) o;
         return Objects.equals(hiveTableName, other.hiveTableName) &&
-                Objects.equals(partitionValues, other.partitionValues);
+                Objects.equals(partitionValues, other.partitionValues) &&
```

Comment:
surprised that partitionName was not there before here in this equals method !

Reply:
partitionValues was derived from partitionName. Hence it was not explicitly used I think.